src/test/java/com/thomashan/collection/immutable/ImmutableListImplTest.java

```diff
@@ -0,0 +1,253 @@
+package com.thomashan.collection.immutable;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many static imports may lead to messy code](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4509131)